aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp

```diff
@@ -59,6 +69,33 @@ void convert_indices_from_coo_to_csr_cpu(const Tensor& result, const Tensor& inp
     data_out[i] = static_cast<output_t>(numel);
 }
 
+template <typename input_t, typename output_t>
+void convert_indices_from_csr_to_coo_cpu(const Tensor& result, const Tensor& crow_indices, const Tensor& col_indices, const int64_t size) {
+  int64_t numel = crow_indices.numel();
+  const input_t* col_indices_data_in = col_indices.data_ptr<input_t>();
+  const input_t* crow_indices_data_in;
+  if (crow_indices.is_contiguous()) {
+    crow_indices_data_in = crow_indices.data_ptr<input_t>();
+  } else {
+    auto crow_indices_contiguous = crow_indices.contiguous();
+    crow_indices_data_in = crow_indices_contiguous.data_ptr<input_t>();
+  }
```

Comment:
Was about to write that using `tensor.contiguous().data_ptr<>` creates a dangling pointer as `.contiguous()` creates a new Tensor which goes out of scope before `.data_ptr` is called on it.
Current version looks better but can be simplified
```suggestion
  auto crow_indices_ = crow_indices.expect_contiguous();
  const input_t* crow_indices_data_in = crow_indices_->data_ptr<input_t>();
```

Reply:
Nice! Wasn't aware of `expect_contiguous`.